### ISSUE TITLE
Screen readers doesn't read grid summaries (Sum, count) on dxDataDrid since they don't get focused

### ISSUE THREAD

**Issue by RezaRahmati** [DevExpress/DevExtreme#11748]
### Steps to Reproduce:

1. Open this sample [https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/GridSummaries/Angular/Light/](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/GridSummaries/Angular/Light/)
2. Open a screen reader such as JAWS
3. Start pressing Tab until end of data
4. Press another Tab

### Results You Received:
It jump to Paging

### Results You Expected:
It Jump to summaries so screen reader will read that

### Environment Details:
Devextreme: 19.1.6

**Comment by vconst** [comment]
Hi @RezaRahmati,

Currently, the summary cells are not accessible in DataGrid. We are going to support accessibility for summary cells in future versions.

**Comment by RezaRahmati** [comment]
thanks, would you please share the ticket link so I can watch it

**Comment by onebalaban** [comment]
Hello,

We don't have a ticket for this task. I suggest you follow us in our [blogs](https://js.devexpress.com/Blogs/) and GitHub for further news.
